L1Trigger/L1CaloTrigger/plugins/Phase2L1CaloJetEmulator.cc

```diff
@@ -453,12 +655,271 @@ void Phase2L1CaloJetEmulator::produce(edm::Event& iEvent, const edm::EventSetup&
   iEvent.put(std::move(jetCands), "GCTJet");
 }
 
+// Apply calibrations to HCAL energy based on Jet Eta, Jet pT
+float Phase2L1CaloJetEmulator::get_jet_pt_calibration(float& jet_pt, float& jet_eta) const {
```

Comment:
I think you can safely make these constant references:

```c++
float Phase2L1CaloJetEmulator::get_jet_pt_calibration(const float& jet_pt, const float& jet_eta) const {
```

Reply:
Done